app/models/hardware.rb

```diff
@@ -70,7 +70,7 @@ def self.add_elements(parent, xmlNode)
       begin
         parent.hardware.send("m_#{e.name}", parent, e, deletes) if parent.hardware.respond_to?("m_#{e.name}")
       rescue => err
-        _log.warn "#{err}"
+        _log.warn err.to_s
```

Comment:
Well, slight correction...it seems logger calls .inspect, so .to_s may be needed, but case by case.